src/internal.rs

```diff
@@ -155,27 +158,30 @@ impl BitDepthDependentType for Grain {
 #[repr(C)]
 pub(crate) struct TaskThreadData_delayed_fg {
     pub exec: c_int,
-    pub cond: pthread_cond_t,
     pub in_0: *const Rav1dPicture,
     pub out: *mut Rav1dPicture,
     pub type_0: TaskType,
-    pub progress: [atomic_int; 2], /* [0]=started, [1]=completed */
     pub grain: BitDepthUnion<Grain>,
 }
 
+pub(crate) struct TaskThreadData_inner {
+    pub delayed_fg: TaskThreadData_delayed_fg,
+}
```

Comment:
For new types we can use `TitleCase`.

Reply:
Also, what's the point of `TaskThreadData_inner` vs. just using `TaskThreadData_delayed_fg`?

Reply:
It originally had the `cur` field as well and I had to remove it. I'll clean this up, good point.